graphs/adapter-declaration/src/main/kotlin/com/anaplan/engineering/azuki/graphs/adapter/declaration/GraphDeclarationState.kt

```diff
@@ -2,32 +2,54 @@ package com.anaplan.engineering.azuki.graphs.adapter.declaration
 
 import com.anaplan.engineering.azuki.declaration.DeclarationState
 import com.anaplan.engineering.azuki.declaration.DeclarationStateFactory
-import com.anaplan.engineering.azuki.graphs.adapter.declaration.declaration.GraphDeclaration
+import com.anaplan.engineering.azuki.graphs.adapter.declaration.declaration.declareDirectedGraph
+import com.anaplan.engineering.azuki.graphs.adapter.declaration.declaration.declareUndirectedGraph
 
 class GraphDeclarationState : DeclarationState() {
 
     fun declareGraph(graphName: String) {
         checkForDuplicate(graphName)
-        declarations[graphName] = GraphDeclaration<Any>(graphName, standalone = true)
+        declarations[graphName] = declareUndirectedGraph<Any>(graphName, standalone = true)
```

Comment:
🚫 What I said in https://github.com/anaplan-engineering/azuki/pull/40#discussion_r1576042326 was that the _function_ name (i.e. two lines above) has to change from `declareGraph` to `declareUndirectedGraph`. I just couldn't anchor the comment there because there were no changes there...

Reply:
Yeah looking back at it I understand that now. My bad.